registry/access_control/db_rbac.py

```diff
@@ -0,0 +1,110 @@
+from access_control import config
+from common.database import connect
+from access_control.models import AccessType, UserRole, RoleType, SUPER_ADMIN_SCOPE
+from access_control.interface import RBAC
+import os
+import logging
+
+class DbRBAC(RBAC):
+    def __init__(self):
+        if not os.environ.get("CONNECTION_STR"):
+            os.environ["CONNECTION_STR"] = config.CONNECTION_STR
+        self.conn = connect()
+        # cached user role records
+        self._refresh_cache()
+
+    def _refresh_cache(self):
+        #TODO: add user role cache refresh interval, e.g. daily
```

Comment:
> #TODO: add user role cache refresh interval, e.g. daily [](http://example.com/codeflow?start=7&length=56)

Do we really need this TODO?

Reply:
This is TODO is to catch up the manual data insert into the `userroles` SQL table. 
We could do regular refresh as well as a manual refresh button on UI. What do you think?

Reply:
Cache doesn't work when RBAC service is in a cluster. And manually updating database is **not** a supported scenario.
We can skip cache and read/write database directly for now to make sure every operation is correct. Let's deal with performance later.

Reply:
Updated. Now every operation will read from database rather than refer to a cache